pulsar-client/src/test/java/org/apache/pulsar/client/impl/MessageImplTest.java

```diff
@@ -425,7 +425,7 @@ public void testTypedSchemaGetNullValue() {
         assertNull(msg.getValue());
     }
 
-    @Test(timeOut = 30000)
+    @Test(timeOut = 300000)
```

Comment:
Why did you increase timeout from 30 seconds to 300 seconds?

Reply:
Increase this for debugging.  Reset to 30s.